photosynthesis_metrics/msid.py

```diff
@@ -0,0 +1,406 @@
+r"""Implemetation of Multi-scale Evaluation metric, based on paper
+ https://arxiv.org/abs/1905.11141 and author's repository https://github.com/xgfs/msid
+"""
+from functools import partial
+from typing import List, Tuple, Optional, Callable
```

Comment:
Remove unused imports
```suggestion
from typing import List, Tuple
```